Documentation/ExtensionArchitecture/FileStructure/ComposerJson.rst

```diff
@@ -1,155 +1,107 @@
-.. include:: /Includes.rst.txt
-.. index::
-   File; EXT:{extkey}/composer.json
-   Extension development; composer.json
+..  include:: /Includes.rst.txt
+..  index::
+    File; EXT:{extkey}/composer.json
+    Extension development; composer.json
 
-.. _composer-json:
-.. _files-composer-json:
+..  _composer-json:
+..  _files-composer-json:
 
-=======================================
+=====================
 :file:`composer.json`
-=======================================
+=====================
 
 *-- required* in Composer-based installations
 
-.. note::
+..  contents::
+    :local:
+
 
-   While the file :file:`composer.json` is currently not strictly required
-   for an extension to function properly in legacy non-Composer installations
-   it is recommended to keep it in any public extension that is published to
-   TYPO3 Extension Repository (TER).
+Introduction
+============
 
-Including a :file:`composer.json` is strongly recommended for a number of reasons:
+`Composer <https://getcomposer.org/>`__ is a tool for dependency management in
+PHP. It allows you to declare the libraries your extension depends on and it
+will manage (install/update) them for you.
+
+..  note::
+    While the file :file:`composer.json` is currently not strictly required
```

Comment:
Is that true? I think it was made mandatory with v11 or v12?

Reply:
Good point. Have to check.

Reply:
Nope, found nothing in the breaking changelogs of v11 and v12. It has also no value at the current time as ext_emconf.php is still necessary in legacy installations.